awx/network_ui/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+
+class Device(models.Model):
+
+    device_id = models.AutoField(primary_key=True,)
+    topology = models.ForeignKey('Topology',)
+    name = models.CharField(max_length=200, blank=True)
+    x = models.IntegerField()
+    y = models.IntegerField()
+    id = models.IntegerField()
```

Comment:
How is this different from `device_id`, could we unify these into the same field?

Reply:
In general I'd like to have the pk on the models called `id` to stay consistent with our other models.

Reply:
I've changed the pk to id on all the models.